src/x86/avx.rs

```diff
@@ -126,9 +176,31 @@ extern "C" {
     fn addsubps256(a: f32x8, b: f32x8) -> f32x8;
     #[link_name = "llvm.x86.avx.round.pd.256"]
     fn roundpd256(a: f64x4, b: i32) -> f64x4;
+    #[link_name = "llvm.x86.avx.round.ps.256"]
+    fn roundps256(a: f32x8, b: i32) -> f32x8;
+    #[link_name = "llvm.x86.avx.sqrt.pd.256"]
+    fn sqrtpd256(a: f64x4) -> f64x4;
+    #[link_name = "llvm.x86.avx.sqrt.ps.256"]
+    fn sqrtps256(a: f32x8) -> f32x8;
 }
 
+// TODO: Remove once a macro is ipmlemented to automate these tests
+// https://github.com/rust-lang-nursery/stdsimd/issues/49
 #[cfg(test)]
+#[cfg_attr(test, assert_instr(vroundps))]
+fn test_mm256_round_ps(a: f32x8) -> f32x8 {
```

Comment:
I think this is missing `#[target_feature]`? 

(and the one below)

could these also move next to the functions they're testing? (hopefully temporarily). With https://github.com/rust-lang-nursery/stdsimd/pull/57 this shouldn't even be necessary to have a shim!